python/baseline/reader.py

```diff
@@ -335,7 +335,7 @@ def load(self, filename, vocabs, batchsz, shuffle=False, sort_key=None):
         for i, example_tokens in enumerate(texts):
             example = {}
             for k, vectorizer in self.vectorizers.items():
-                example[k], lengths = vectorizer.run(example_tokens, vocabs[k])
+                example[k], lengths = vectorizer.run(example_tokens, vocabs.get(k))
```

Comment:
This is a change to facilitate the idea that a feature might just be passed to the example dict instead of being passed through an embeddeding